src/components/Oracles/MarchMadness/TeamStatsViewer.tsx

```diff
@@ -0,0 +1,194 @@
+import {
+  Player,
+  StatisticsData,
+  PlayerAverage,
+  Total,
+} from '@/types/Statistics'
+import {
+  Accordion,
+  AccordionButton,
+  AccordionItem,
+  AccordionPanel,
+  Box,
+  Flex,
+  Icon,
+  SimpleGrid,
+  Table,
+  TableContainer,
+  Tbody,
+  Td,
+  Text,
+  Th,
+  Thead,
+  Tr,
+} from '@chakra-ui/react'
+import { TableRowData } from '@/data/MarchMadnessTableData'
+import React, { memo, useMemo } from 'react'
+import { HiOutlinePlusCircle, HiOutlineMinusCircle } from 'react-icons/hi'
+
+const TeamStatsTable = ({
+  full_name,
+  average,
+  total,
+}: Pick<Player, 'full_name' | 'total' | 'average'>) => {
+  const tableRows = useMemo(
+    () => new Set([...Object.keys(average), ...Object.keys(total)]),
+    [],
```

Comment:
The dependency array needs average and total so tableRows gets updated again once the component renders with new average or total info, the warning should be fixed after including them.

Reply:
Okay, I'll push the fix.